docs/builder/manifest.md

```diff
@@ -316,4 +316,163 @@ If you only want to provide a single icon, your icons array could also look like
+* `icons`: A list of `icon` objects that will show when given the option to open the given file with your app.
+* `launch_type`: Dictates the amount of app instances launch when handling multiple files at once. `single-client` is the default but this field can also be set to `multiple-clients` if you want an app instance to launch for each file  being handled.
+
+```json
+"file_handlers": [
+    {
+      "action": "/open-pdf",
+      "accept": {
+        "application/pdf": [".pdf"]
+      },
+      "icons": [
+        {
+          "src": "pdf-icon.png",
+          "sizes": "256x256",
+          "type": "image/png"
+        }
+      ],
+      "launch_type": "single-client"
+    },
+    {
+      "action": "/show-model",
+      "accept": {
+        "application/sla": ".stl",
+        "application/octet-stream": ".fbx"
+      },
+      "icons": [
+        {
+          "src": "3d-printer-icon.png",
+          "sizes": "256x256",
+          "type": "image/png"
+        }
+      ],
+      "launch_type": "multiple-clients"
+    }
+
+]
+```
+
+### launch_handlers: `string` | `Array`
+
+`launch_handlers` is an optional member that controls the launch of a web application. It's singular value `client_mode` can take on the following values:
+
+* `auto`: the user agent makes the decision based on the context.
+* `focus-existing`: If the web app is already open, it is brought into focus without navigating to the launch target URL.
+* `navigate-existing`: If the web app is already open, it is brought into focus and naviagates to the URL made available by `Window.launchQueue`
+* `navigate-new`: A new instance of the web app is opened and it navigates to the URL made available by `Window.launchQueue`
+
+In the second example below, if `navigate-existing` is unavailable it will fallback to the next value in the list.
+
+```json
+"launch_handler": {
+    "client_mode": "navigate-existing"
+}
+
+"launch_handler": {
+    "client_mode": ["navigate-existing, auto"]
+}
+```
+
+?> In any instance where the app is not already running, `navigate-new` will be used instead.
+
+### handle_links: `string`
+
+`handle_links` is an optional member that specifies the default link handling for the web app. It can take on the following values:
+
+* `auto`: The user agent should select the appropriate behavior for the platform (Default if not otherwise specified).
+* `preferred`: the user agent should open in-scope links within the installed application.
+* `not-preferred`: The user agent should not open links within the installed application.
+
+```json
+"handle_links": "preferred"
+```
+
+?> In any instance where the app is not already running, `navigate-new` will be used instead.
+
+### scope_extensions: `Array`
+
+`scope_extensions` is an optional member that specifies a list of origin patterns to associate with. This allows for your app to control multiple subdomains and top level domains to behave as one.
```

Comment:
```suggestion
`scope_extensions` is an optional member that specifies a list of origin patterns to associate with. This allows for your app to control multiple subdomains and for top-level domains to behave as one.
```

Reply:
The sentence I was trying to reword here was 

> Use case/goal: Allow sites that control multiple subdomains and top level domains to behave as one contiguous web app.

Let me know if your change still makes sense with this context

Reply:
https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md#use-cases--goals